data/transactions/logic/doc.go

```diff
@@ -191,13 +191,13 @@ var opDocByName = map[string]string{
 	"itxn_field":  "set field F of the current inner transaction to A",
 	"itxn_submit": "execute the current inner transaction group. Fail if executing this group would exceed the inner transaction limit, or if any transaction in the group fails.",
 
-	"box_create":  "make a box",
-	"box_extract": "read from a box",
-	"box_replace": "write to a box",
-	"box_del":     "delete a box",
-	"box_len":     "length of a box",
-	"box_get":     "full contents of a box",
-	"box_put":     "write contents of box",
+	"box_create":  "create a box named A, of length B. Fail if A is empty or B exceeds 32,384. Returns 0 if A already existed, else 1",
```

Comment:
Given the discussion on the difference in the box'es **value** depending on whether the box existed prior to `box_create` I recommend some wording specifying this behavior.

Reply:
Added this to the extra doc:
```
"box_create": "Newly created boxes are filled with 0 bytes. Boxes are unchanged by `box_create` if they already exist.",
```